book/installation.rst

```diff
@@ -47,21 +44,21 @@ Open your command console and execute the following command:
 
 .. code-block:: bash
 
-    c:\> php -r "readfile('http://symfony.com/installer');" > symfony.phar
+    c:\> php -r "readfile('http://symfony.com/installer');" > symfony
 
-Then, move the downloaded ``symfony.phar`` file to your projects directory and
+Then, move the downloaded ``symfony`` file to your projects directory and
```

Comment:
Here there are some instructions proposed by a Symfony Installer user (https://github.com/symfony/symfony-installer/issues/117) but I haven't tested them. Ideally we'd provide an executable installer as Composer does.